Mastodon/Scene/Profile/Header/View/ProfileHeaderView.swift

```diff
@@ -238,20 +235,14 @@ final class ProfileHeaderView: UIView {
         return metaText
     }()
     
-    override init(frame: CGRect) {
-        super.init(frame: frame)
-        _init()
-    }
-    
-    required init?(coder: NSCoder) {
-        super.init(coder: coder)
-        _init()
-    }
-    
-}
+    init(account: Mastodon.Entity.Account, me: Mastodon.Entity.Account, relationship: Mastodon.Entity.Relationship?) {
+
+        viewModel = ViewModel(account: account, me: me, relationship: relationship)
```

Comment:
I would probably construct the `ProfileHeaderView` with the ViewModel here to only have the three params in the ViewModel's constructor.

Reply:
I keep thinking: Why not just using the `ProfileViewModel` or use something similar to your `MastodonStatus`?